openedx/core/djangoapps/content_libraries/permissions.py

```diff
@@ -87,7 +252,9 @@ def is_course_creator(user):
     is_global_staff |
     # Libraries with "public read" permissions can be accessed only by course creators
     (Attribute('allow_public_read', True) & is_course_creator) |
-    # Otherwise the user must be part of the library's team
+    # Users can access libraries within their authorized scope (via Casbin/role-based permissions)
+    HasPermissionInContentLibraryScope("view_library") |
```

Comment:
Should change this to a constant from openedx-authz: https://github.com/openedx/openedx-authz/blob/main/openedx_authz/constants/permissions.py